src/test/resources/io/fabric8/elasticsearch/plugin/searchguard_roles_acl.yml

```diff
@@ -13,7 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+sg_unrecognized_child:
```

Comment:
how is this test entry used in our tests?

Reply:
Tests a NullPointer I discovered while fixing this issue https://github.com/fabric8io/openshift-elasticsearch-plugin/blob/2.4.4/src/test/java/io/fabric8/elasticsearch/plugin/acl/SearchGuardRoleACLTest.java#L158